src/views/profile.hbs

```diff
@@ -0,0 +1,14 @@
+<h3>{{name}} profile</h3>
+<form action="/logout">
+<input type="submit" value="Log out">
+</form>
+<h1>TutorialsApp</h1>
+
+
+<form action="/addtutorial" method="POST">
+  <input type="text" name="language_name" value="" placeholder="Add language name..">
+  <input type="text" name="link" value="" placeholder="Add link..">
+   <input type="submit" name="" value="Add">
+</form>
+
+<p>{{message}}</p>
```

Comment:
Add the js function that you render the message here

Reply:
this one for add tutorial function